app/views/works/_edit_javascript.html.erb

```diff
@@ -48,18 +48,23 @@
 
     var deleteCreator = function(num) {
       var rowToDelete = `#creator_row_${num}`;
+      var rowExists = $(rowToDelete).length > 0;
       var orcidId = `#orcid_${num}`;
       var givenNameId = `#given_name_${num}`;
       var familyNameId = `#family_name_${num}`;
       var name = $(orcidId).val() + " " + $(givenNameId).val() + " " + $(familyNameId).val();
-      var emptyName = (name === undefined) || (name.trim().length == 0);
-      if (emptyName) {
-        // delete it without asking
-        $(rowToDelete).remove();
-      } else {
-        if (confirm(`Remove creator ${name}`)) {
+      var emptyName = (name.trim().length == 0);
+      if (rowExists) {
+        if (emptyName) {
+          // delete it without asking
           $(rowToDelete).remove();
+        } else {
+          if (confirm(`Remove creator ${name}`)) {
+            $(rowToDelete).remove();
+          }
         }
+      } else {
+        console.log(`${rowToDelete} does not exist - nothing to do`);
```

Comment:
I have no objection to leaving this here, but I just wish to ensure that this was not used solely for debugging.